docs/api.rst

```diff
@@ -1457,6 +1429,81 @@ to handle it, which defaults to print a traceback and ignoring the exception.
     :param payload: The event's data.
     :type payload: :class:`AutoModActionExecutionEvent`
 
+Other Events Reference
+~~~~~~~~~~~~~~~~~~~~~~
+.. function:: on_interaction(interaction)
```

Comment:
This and on_application_command (as well as other things like autocomplete that aren't documented) can be put in an 'interactions' category